profiles/restrictive

```diff
@@ -622,6 +622,9 @@ org.freedesktop.fwupd.modify-config                             auth_admin:no:au
 # fwupd (bsc#1193310)
 org.freedesktop.fwupd.downgrade-internal-trusted                auth_admin:no:auth_admin_keep
 org.freedesktop.fwupd.downgrade-hotplug-trusted                 auth_admin:no:auth_admin_keep
+# fwupd (bsc#1204026)
+org.freedesktop.fwupd.get-bios-settings                         auth_admin:no:auth_admin_keep
```

Comment:
Nitpick: do we want to have `auth_admin_keep` in restrictive?

Reply:
I explicitly decided to keep it. The getter info is not that critical (in fact upstream even allows to get most of this data without *any* authentication). Then, tightening upstream settings from _keep to non-keep have a habit to break user experiences, because of needlessly re-occoring popups.

Reply:
Fair enough